pkg/operator/ceph/csi/spec.go

```diff
@@ -533,10 +534,11 @@ func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset
 			if err != nil {
 				return false, errors.Wrapf(err, "failed to apply multus configuration to CephCluster %q", cephCluster.Name)
 			}
+			isMultusApplied = true
 		}
 	}
 
-	return true, nil
+	return isMultusApplied, nil
```

Comment:
@leseb do we need this bool ? or cant we just survive with 'error' return in caller for nil , or error ?  :)

Reply:
The bool checks whether multus is applied and error checks for any underlaying error so I think it's cleaner this way :)